utopian-io/v2.utopian.io
# Search bounties page

### Verify that
Git checklist

Front end checklist

API checklist

Tests and comments checklist

### My PR doesn't concern a Jira task
Search bounties page

---

Please provide a test plan so that the reviewers can see how you tested your work and/or the different acceptance criteria described in the task.

### **Test plan**

_Must work the same way as the article search page_

* Search something and click on 'Bounties' on the left menu bar or go to http://localhost:8080/en/search/bounties 
* Search for the title and use the filters


**Expected result:** You should ...
